packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -17,6 +17,12 @@ export class LocalChannelStorageService implements IChannelStorageService {
         return contents !== undefined ? Promise.resolve(contents) : Promise.reject(new Error("Not found"));
     }
 
+    public async readBlob(path: string): Promise<ArrayBufferLike> {
+        const blob = this.readBlobSyncInternal(path, this.tree);
+        return blob !== undefined ? stringToBuffer(blob.contents, blob.encoding) :
```

Comment:
I think we prefer throw vs. rejected promises (it's the same at the end, so it's more stylistic question).

if (blob === undefined) { throw ... ; }
return stringToBuffer(blob.contents, blob.encoding);